utilities/openemr-env-installer/openemr-env-installer

```diff
@@ -382,7 +420,8 @@ startup_the_env() {
 	echo -e "\033[34m==>\033[0mOr run the below command to startup \033[33minsane env\033[0m."
 	echo -e "\033[32m   cd ${code_location}/openemr/contrib/util/docker && openemr-cmd up\033[0m"
 	echo -e "\033[34m==>\033[0mOr run the below command to startup \033[33mOpenEMR Orchestration\033[0m."
-	echo -e "\033[32m   minikube start\033[0m"
+	echo -e "\033[33m   Please run with '--driver=none' for minikube if run as root.\033[0m"
+	echo -e "\033[32m   minikube start --driver=docker\033[0m"
```

Comment:
Which driver to use will likely be dependent on the actual environment.
The `--driver=none` means it will install everything on the current os (won't separate it on a vm or docker), so could be problematic for users that do not want that. I didn't even know about the docker driver, which plan to try (I have been using the virtualbox driver).

Reply:
@bradymiller, yep, because there is some issue about cannot detect the driver. So I will move the none option, and mentioned the docker driver